generation/new_client/README.md

```diff
@@ -172,17 +172,23 @@ Example: `https://cloud.google.com/alloydb/docs`
 
 ### REST Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+Not every library has these, as they are generated adhoc. Poke around the product docs link to see if there is a valid REST API reference link.
```

Comment:
Would you avoid starting a section with a negative statement? (It sounds like similar to starting with a section with "Note that ...")